packages/react-jsx-highcharts/src/components/PlotBandLine/UsePlotBandLineLifecycle.js

```diff
@@ -25,8 +25,17 @@ export default function usePlotBandLineLifecycle(props, plotType) {
       };
 
       if (plotbandline) axis.removePlotBandOrLine(idRef.current);
+      axis.addPlotBandOrLine(opts, plotType);
       setPlotbandline({
-        object: axis.addPlotBandOrLine(opts, plotType)
+        id: myId,
+        getPlotBandLine: () => {
+          if (axis && axis.object && axis.object.plotLinesAndBands) {
+            const plotbandlineObject = axis.object.plotLinesAndBands.find(
+              plb => plb.id === myId
+            );
+            return plotbandlineObject;
+          }
+        }
```

Comment:
This is technically a breaking change as it changes the object returned by usePlotBandLine.

 But then again it did not work at all before.

Reply:
Adding object getter might actually work  here instead.